plugins/calendar/src/index.ts

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import type { Account, Class, Doc, DocumentMapper, Ref, Space } from '@anticrm/core'
+import { AnyComponent } from '@anticrm/ui'
+import { DerivedData } from '@anticrm/core'
+import type { Plugin, Service } from '@anticrm/platform'
+import { plugin } from '@anticrm/platform'
+import type { Asset, IntlString, Resource } from '@anticrm/status'
+
+
+export interface Calendar extends Space {}
+
+export interface Event extends Doc {
+  name: string
+  description: string
+  startsAt: Date
+  endsAt: Date
+  participants: Array<Ref<Account>>
```

Comment:
Please use eslint config as in `@anticrm/core` in this case it will have eslint rule configured to use Ref<Account>[]

Reply:
For some reason eslintrc update hasn't affected this line

Reply:
Will check